daemon/redis.c

```diff
@@ -2043,21 +1553,15 @@ static void restore_thread(void *call_p, void *ctx_p) {
 	redisReply *call = call_p;
 	struct redis *r;
 	str callid;
+	str_init(&callid,call->str);
```

Comment:
use `str_init_len(&callid, call->str, call->len);` (as was removed below) to avoid an unnecessary `strlen()`